resolwe/flow/expression_engines/jinja/filters.py

```diff
@@ -36,6 +38,17 @@ def slug(data):
     return apply_filter_list(lambda datum: _get_data_attr(datum, 'slug'), data)
 
 
+def input_(data, field_path):
+    """Return a hydrated value of the ``input`` field."""
+    data_obj = Data.objects.get(id=data['__id'])
+
+    inputs = copy.deepcopy(data_obj.input)
+    hydrate_input_references(inputs, data_obj.process.input_schema)
```

Comment:
This can be optimized by hydrating only the required field. But this would require a major refactoring. So just add a comment for now.

Reply:
Added